internal/state/store.go

```diff
@@ -29,10 +29,11 @@ const (
 
 const (
 	// prefixes are unique across all tm db's
-	prefixValidators      = int64(5)
-	prefixConsensusParams = int64(6)
-	prefixABCIResponses   = int64(7)
-	prefixState           = int64(8)
+	prefixValidators       = int64(5)
+	prefixConsensusParams  = int64(6)
+	prefixAllABCIResponses = int64(7)
+	prefixState            = int64(8)
+	prefixLastABCIResponse = int64(13)
```

Comment:
I don't understand why the constants start a 5 here, nor why it jumps from 8 to 13.

Should we assume other readers understand this layout? Would it be valid to include an explanatory comment here?

Reply:
This is because we try to keep unique prefixes across _all_ tm databases i.e. keys 1 - 4 are for `blockstore.db`. We don't necessarily have to do this - but would make things easier if we were to merge state store and blockstore or if a user has decides to put all the stores within the same database